app/src/main/java/com/omelan/cofi/pages/RecipeEdit.kt

```diff
@@ -158,121 +160,126 @@ fun RecipeEdit(
             )
         }
     ) {
-        BoxWithConstraints {
-            LazyColumn(
-                modifier = Modifier
-                    .fillMaxWidth()
-                    .fillMaxHeight()
-                    .background(color = MaterialTheme.colorScheme.background),
-                contentPadding = PaddingValues(
-                    bottom = maxHeight / 2,
-                    top = Spacing.big,
-                    start = Spacing.big,
-                    end = Spacing.big,
-                ),
-            ) {
-                item {
-                    Row(verticalAlignment = Alignment.CenterVertically) {
-                        IconButton(
-                            onClick = {
-                                coroutineScope.launch {
-                                    if (bottomSheetScaffoldState.bottomSheetState.isExpanded) {
-                                        bottomSheetScaffoldState.bottomSheetState.collapse()
-                                    } else {
-                                        bottomSheetScaffoldState.bottomSheetState.expand()
+        CompositionLocalProvider(LocalTextSelectionColors provides textSelectionColors) {
```

Comment:
I feel like it should be moved up to `MainActivity.kt` to make sure that it's propagated to all views in the app. What do you think?

Reply:
Yea, I felt like this too. There are two things that have an impact on my desicion:

1. If I remember correctly CompositionLocalProviders should be added as deep as possible if not used often. Obviously it sould be added in MainActivity if you are using it multiple times. Currently its only used once since there are no other inputs.
2. It may be wise to move it up to MainAcitivity anyways to prevent confusion and bugs in the future.

What do you think? Should I move it to MainActivity? I think thats feasible.

Reply:
I've skimmed CompositionLocal documentation and I think that you were right the first time. If it ever becomes a common occurrence across the code then I'll think of refactoring that.